src/components/VaccinationsTable/VaccinationsTable.style.tsx

```diff
@@ -81,6 +81,7 @@ export const ColumnHeader = styled.p`
   color: ${COLOR_MAP.GRAY_BODY_COPY};
   margin: 0 0 1rem;
   text-align: center;
+  font-size: 0.875rem;
```

Comment:
not sure why this snuck into this PR, but I assume it's intentional. 😁

Reply:
yeah, the column headers are 16px in the mocks but use `font-variant: small-caps`, so im dropping the font size to account for that